backend/prisma/schema.prisma

```diff
@@ -70,6 +71,8 @@ model Plan {
   createdAt        DateTime         @default(now())
   updatedAt        DateTime         @updatedAt
   PlanMembership   PlanMembership[]
+  Contact          Contact[]
```

Comment:
is this field the same as `contacts`?

Reply:
not sure what you mean?

Reply:
should I change the capital Contact in the Plan model to lowercase to match the users model? We should prob establish how we want to capitalize the "label" in our db models.

Reply:
The convention is camel case, so lets use `contacts` here

Reply:
done!